authorize.go

```diff
@@ -63,7 +64,11 @@ func (a *autographer) authorizeHeader(r *http.Request) (auth *hawk.Auth, userid
 	if err != nil {
 		return nil, "", err
 	}
-	hawk.MaxTimestampSkew = a.auths[userid].hawkMaxTimestampSkew
+	hawkSkewAuth, err := a.getAuthByID(userid)
+	if err != nil {
+		return nil, "", errors.Wrapf(err, "error finding auth for id %s for hawk.MaxTimestampSkew", userid)
+	}
+	hawk.MaxTimestampSkew = hawkSkewAuth.hawkMaxTimestampSkew
```

Comment:
this doesn't need to be fixed here, but I somewhat question the value of setting the skew parameter differently for each signer vs having a global one for autograph.

Reply:
yeah captured that in https://github.com/mozilla-services/autograph/issues/414